src/pudl_archiver/archivers/epacems.py

```diff
@@ -73,7 +73,9 @@ async def get_year_quarter_resource(
 
         # Create zipfile to store year/quarter combinations of files
         filename = f"epacems-{year}-{quarter}.csv"
+        # filename = f"epacems-{year}q{quarter}.csv"
         archive_path = self.download_directory / f"epacems-{year}-{quarter}.zip"
+        # archive_path = self.download_directory / f"epacems-{year}q{quarter}.zip"
```

Comment:
I left these commented out file name and path updates because in an ideal world we would convert all of the file names to be more consistent with the partition. this is not a required change because the datastore knows how to grab a file based on the partition regardless of if the partition and the filename match because of the info stored in the datapackage.json. Nonetheless this would still be a good change to make if we could easily fully update the file names.

Reply:
I think these comments as-is would be kind of confusing in the future, since they don't have context as to why they're commented out.

When we make the next quarterly updates to the archive, we'll need to re-download and re-upload all the files anyways, right? Could we rename everything then? If we can, then I think we should probably go ahead and change the code now.

Reply:
yeah this is a very good point! I guess I am partial to adding a comment in there with a `# TODO:` and creating an issue once we talk about how to make re-making this archive a less flaky thing to do.